src/batch/client.ts

```diff
@@ -149,37 +131,28 @@ export class BatchTranscription {
       15 * 60 * 1000, // 15 minutes timeout
     );
 
-    return await this.getJobResult(submitResponse.id, format);
+    return await this.getJobResult(submitResponse.id, format || 'json-v2');
   }
 
-  async createJob({
-    input,
-    fileName,
-    transcription_config,
-    translation_config,
-    output_config,
-    summarization_config,
-    auto_chapters_config,
-    topic_detection_config,
-  }: CreateJobConfig): Promise<CreateJobResponse> {
+  async createJob(
+    input: JobInput,
+    jobConfig: Omit<JobConfig, 'type'>,
+  ): Promise<CreateJobResponse> {
     if (this.config.apiKey === undefined)
       throw new Error('Error: apiKey is undefined');
 
-    const config: JobConfig = {
+    const config = {
       type: 'transcription',
```

Comment:
would type ever be anything other than transcription? If so, would this function be better called `createTranscriptionJob`?

Reply:
Yeah, you can also pass `type: "alignment"` with an associated `alignment_config`. I think it didn't make it into the SDK because we don't expose it in the portal.

I think I will rename it to `createTranscriptionJob`, which will give us room to implement `createAlignmentJob` as well at some point.